lms/static/sass/discussion/_discussion.scss

```diff
@@ -784,13 +475,13 @@ body.discussion {
           overflow-y: auto;
           max-height: 600px;
 
+          //TO-DO: review / remove this from inline styling case 
```

Comment:
Author To-Do: Review inline styling module to ensure there are no visual or interaction regressions. Noted in PR checklist, will remove this note after that testing as well.